sphinx/source/docs/user_guide/quickstart.rst

```diff
@@ -145,26 +145,28 @@ plot as shown below:
 Jupyter Notebooks
 =================
 
-At this point we should mention Jupyter (formerly IPython) notebooks.
```

Comment:
I personally find non-reference docs that are somewhat conversational in tone much more approachable, is there rationale or research supporting one way or the other? If not I'd rather leave this sentence.